Fortinet/fortiproxy/ingest/parser.yml

```
@@ -98,6 +98,10 @@ stages:
           observer.product: FortiProxy
           observer.hostname: "{{parsed_event.message.devname}}"
 
+      - set:
+          fortinet.referralurl: "{{parsed_event.message.referralurl}}"
```

Is referralurl corresponding to the referrer in HTTP?
In this case, [`http.request.referrer`](https://www.elastic.co/guide/en/ecs/current/ecs-http.html#field-http-request-referrer) seems to be more suitable:
```suggestion
          http.request.referrer: "{{parsed_event.message.referralurl}}"
```

---

Ok, I change that